app/views/admin/absences/show.html.slim

```diff
@@ -0,0 +1,7 @@
+.container
+  .row
+    .col-md-12
+        h1 = "Absence Request for #{@user.name}"
```

Comment:
Indentation a little off here (I think it should be two spaces instead of four from the `.col-md-12` above).